_component_design/switches.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: Switches
-parent: null
+parent: Inputs and Controls
```

Comment:
Inputs and controls* (use sentence style capitalisation)

Reply:
Has to be this way due to back end reasons or it won't appear on the Inputs and Controls family page. Good news is this is only used as system data and doesn't appear like this on any content.